### ISSUE TITLE
Exporting too many nodes fails with message "Filesize would require ZIP64 extensions"

### ISSUE THREAD

**Issue by nmounet** [aiidateam/aiida_core#905]
Example of output of the `verdi export create` command:

```
STARTING EXPORT...
STORING DATABASE ENTRIES...
Exporting a total of 1626361 db entries, of which 1626358 nodes.
STORING NODE ATTRIBUTES...
STORING NODE LINKS...
STORING GROUP ELEMENTS...
STORING DATA...
Traceback (most recent call last):
  File "/home/aiida/aiidapy/bin/verdi", line 11, in <module>
    sys.exit(run())
  File "/home/aiida/aiidapy/local/lib/python2.7/site-packages/aiida/cmdline/verdilib.py", line 1043, in run
    aiida.cmdline.verdilib.exec_from_cmdline(sys.argv)
  File "/home/aiida/aiidapy/local/lib/python2.7/site-packages/aiida/cmdline/verdilib.py", line 1028, in exec_from_cmdline
    CommandClass.run(*argv[command_position + 1:])
  File "/home/aiida/aiidapy/local/lib/python2.7/site-packages/aiida/cmdline/baseclass.py", line 190, in run
    function_to_call(*args[1:])
  File "/home/aiida/aiidapy/local/lib/python2.7/site-packages/aiida/cmdline/commands/exportfile.py", line 31, in cli
    verdi()
  File "/home/aiida/aiidapy/local/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/aiida/aiidapy/local/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/aiida/aiidapy/local/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/aiida/aiidapy/local/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/aiida/aiidapy/local/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/aiida/aiidapy/local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/aiida/aiidapy/local/lib/python2.7/site-packages/aiida/cmdline/commands/exportfile.py", line 134, in create
    outfile=outfile, overwrite=overwrite, **additional_kwargs
  File "/home/aiida/aiidapy/local/lib/python2.7/site-packages/aiida/orm/importexport.py", line 2173, in export_zip
    export_tree(what, folder=folder, silent=silent, **kwargs)
  File "/home/aiida/aiidapy/local/lib/python2.7/site-packages/aiida/orm/importexport.py", line 1918, in export_tree
    }, f)
  File "/home/aiida/aiidapy/local/lib/python2.7/site-packages/aiida/orm/importexport.py", line 2049, in __exit__
    self.close()
  File "/home/aiida/aiidapy/local/lib/python2.7/site-packages/aiida/orm/importexport.py", line 2041, in close
    self._zipfile.writestr(self._fname, self._buffer.read())
  File "/usr/lib/python2.7/zipfile.py", line 1234, in writestr
    self._writecheck(zinfo)
  File "/usr/lib/python2.7/zipfile.py", line 1114, in _writecheck
    " would require ZIP64 extensions")
zipfile.LargeZipFile: Filesize would require ZIP64 extensions

```

**Comment by nmounet** [comment]
BTW, the "-a" option of `verdi export create` should say what is the default (`zip`).

**Comment by nmounet** [comment]
Some follow-up: the `zip` and `zip-uncompressed ` formats raise the error above, but the `tar.gz` one works fine.

**Comment by sphuber** [comment]
@giovannipizzi should we maybe change the default to use `tar.gz` and add a hint in the doc string of the command about the limitation of the `.zip` extension. Additionally maybe we catch the exception and print a more useful message to tell users to use `tar.gz` if it does hit?

**Comment by giovannipizzi** [comment]
for sure about the better message. For the default switch, I think the reason of using zip is that is was much faster IIRC - can we make a quick benchmark before changing the default when compressing a large number of relatively small files? (probably we can play with the compression level)

**Comment by giovannipizzi** [comment]
Another simple option is to pass the option `allowZip64 = True` (my understanding is that it is activated only if needed, but we should check if it's true, and if it has performance hits). The issue of not being readable by unix standard zip tools is probably not a big issue, if we expect AiiDA to read it, mainly.

**Comment by giovannipizzi** [comment]
This was already solved in #1617
